### PR TITLE
Dispatcher::handle_message does not need Result anymore

### PR DESCRIPTION
**Pull request by phimuemue**
I skimmed the code and saw that `Dispatcher::handle_message` does not need `Result` anymore. It used to, but the relevant code passage referring to `Err` has been removed.

Eliminating this simplifies some places quite a bit.

In case this simplification is unwanted, feel free to discard it, of course.

---
This change is [Reviewable](https://reviewable.io/reviews/graphiteeditor/graphite/365)

### CODE REVIEW

**Comment by TrueDoctor** [comment]
Astute observation, we kinda transitionend towards sending explicit error messages so I think this should work

**Comment by Keavon** [comment]
Lovely, thanks for this simplification! I'm going to wait for @TrueDoctor to give the code review approval.

**Review by TrueDoctor — APPROVED** [review]

**Comment by Keavon** [comment]
Congratulations on your first contribution, @phimuemue! If you have joined our [Discord server](https://discord.graphite.design), I'd be happy to assign you our coveted Code Contributor role if you so wish.

**Pull request merged**
